x-pack/platform/plugins/shared/rule_registry/common/assets/field_maps/technical_rule_field_map.test.ts

```diff
@@ -45,6 +45,11 @@ it('matches snapshot', () => {
         "required": false,
         "type": "keyword",
       },
+      "kibana.alert.attack_ids": Object {
```

Comment:
I guess this also won't be needed, since we decided to move this field to security and attack discovery alerts.